app/src/main/java/com/example/robin/roomwordsample/fragments/NewToDoFragment.kt

```diff
@@ -149,7 +150,17 @@ class NewToDoFragment : Fragment() {
                     WorkManager.getInstance().enqueue(notifyManager)
                 }
                 replyIntent.putExtra("tag", tag)
-                wordViewModel.insert(Word(task, tm, tag, false, description))
+                val colors = resources.getStringArray(R.array.colors)
```

Comment:
Move this variable declaration outside the onCreateView().

Reply:
@rob729 Done :+1: